client/python/ovmsclient/lib/ovmsclient/tfs_compat/http/serving_client.py

```diff
@@ -26,7 +27,8 @@
                                                   HttpModelMetadataResponse,
                                                   HttpPredictResponse)
 
-from ovmsclient.tfs_compat.base.errors import BadResponseError, ModelServerError, raise_from_http
+from ovmsclient.tfs_compat.base.errors import (BadResponseError, ModelServerError,
+                                               raise_from_http, raise_from_http_response)
```

Comment:
This imports seems weird to me on github. Is there any indentation problem here?

Reply:
Now I see it was github :P. Resolved.